package.json

```diff
@@ -2,24 +2,26 @@
   "name": "comapeo-mobile",
   "version": "0.0.1",
   "private": true,
+  "main": "index.js",
   "scripts": {
-    "build:backend": "node ./scripts/build-backend.mjs",
     "android": "npm run build:backend && npm run android-no-backend-rebuild",
-    "android-no-backend-rebuild": "react-native run-android",
-    "ios": "react-native run-ios",
-    "prestart": "npm run build:translations && npm run build:intl-polyfills",
-    "start": "react-native start",
+    "android-no-backend-rebuild": "expo run:android",
+    "ios": "expo run:ios && npm run android-no-backend-rebuild",
+    "prestart": "npm run build:translations && npm run build:intl-polyfills && npm run build:backend",
```

Comment:
Okay, that seems fair. Currently it'll look like that:
- `postinstall` triggers rebuild of all autogenerated files (translations, polyfills, backend),
- `prestart` triggers generation of frontend stuff only (translations, polyfills),
- `android` triggers rebuild of frontend + backend + enables android app,
- `ios` triggers rebuild of frontend + backend + enables ios app,
- `android-no-backend-rebuild`, as name says, just rebuilds frontend and enables android app,
- `ios-no-backend-rebuild` is analogous to android task,
- `build:autogenerated` is just alias for generating translations, polyfills, backend

LMK if we should add some setup notes to readme, as this autogenerated setup is a little bit tricky to understand at first glance.

Reply:
more of a personal preference, but not entirely sure if we need `postinstall` to do all of the work related to autogenerated files. at the moment, the ones in question are only needed for when you want to run the app (or tests once we have more of them) so i'd think that doing that work in `prestart` and before `android` would be okay. i like having `npm install` being as fast as possible, with it mostly being used to deal with setting up the dev environment (not necessarily app code). again, not adamant about this as it's just my personal preference.

otherwise your breakdown seems reasonable to me!

> LMK if we should add some setup notes to readme, as this autogenerated setup is a little bit tricky to understand at first glance.

Think it's okay to not document this for now since things are still changing a bit. The long-term plan is to have a contributors/development-specific doc that details this stuff, which I think would be easier for someone like myself to do :)

Reply:
after some more thinking, think we should remove the `build:autogenerated` script and remove its usage from `postinstall`. think it will make sense to revisit all the scripts setup in a separate PR, but introducing it here unnecessarily increases CI time (more specifically running the `build:backend` command when it's not needed for the CI tasks)